programmers2/서울에서-김서방-찾기.js

```diff
@@ -0,0 +1,10 @@
+function solution(seoul) {
+    let location = "";
+    for(let i=0; i<seoul.length; i++) {
+        if (seoul[i] == "Kim")
+            location = seoul.indexOf(seoul[i]);
+    }
+    var answer = '�輭��� '+ location + '�� �ִ�';
+    return answer;
```

Comment:
이 문제에서는 값을 하나만 찾기에 변수를 따로 선언하지 않는것이 나음
for , if 같은 조건문 뒤에 공백을 붙여야함 
비교는 ===로 해야한다.

if문에서 이미 인덱스를 사용하여 kim을 찾았기에 indexOf는 없어도된다. 

```suggestion
    for (let i = 0; i < seoul.length; i++) {
        if (seoul[i] === "Kim")
            return `김서방은 ${i}에 있다`;
    }

```